### DISCUSSION TITLE
Tasmota 13.1.0 dmp file size

### DISCUSSION THREAD

**Discussion by cruiser2** [arendst/Tasmota#19335] [General]
Hi, just an observation!
Dmp-file size on 13.1.0 = 4.560 Bytes. Decode-config.exe for 13.1.0 expects 4.096 Bytes.

**Comment by arendst** [comment]
It now contains config files from the filesystem too.

Decode-config probably needs a small change to skip the extra bytes. You may try to fool it by removing the extranous bytes over 4096 from the .dmp file.
